src/sentry/db/models/fields/node.py

```diff
@@ -37,14 +37,14 @@ class NodeData(collections.MutableMapping):
         data={...} means, this is an object that should be saved to nodestore.
     """
 
-    def __init__(self, field, id, data=None, wrapper=None):
-        self.field = field
+    def __init__(self, id, data=None, wrapper=None, ref_version=None, ref_func=None):
```

Comment:
`ref_func` is always the same, not sure if it's worth keeping the ability to even pass a custom function but will leave it for now.